### PR TITLE
lgrebe os 5866 add pages for identity and access and application credentials

### PR DESCRIPTION
**Pull request by lgrebe-sys**

### CODE REVIEW

File: pages/04.Reference/13.identity-and-access/docs.en.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+published: true
+date: '20-03-2020 10:40'
+taxonomy:
+    category:
+        - docs
+---
+
+## Overview
+
+OpenStack's Identity API is implemented via Keystone. It provides API client authentication, service discovery, and distributed multi-tenant authorization.
```
Comment (stefreak) [inline pages/04.Reference/13.identity-and-access/docs.en.md]: distributed multi-tenant authorization does not mean anything for our clients, so I would remove that sentence.

File: pages/04.Reference/13.identity-and-access/docs.en.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: Identity and Access
+published: true
+date: '20-03-2020 10:40'
+taxonomy:
+    category:
+        - docs
+---
+
+## Overview
```
Comment (stefreak) [inline pages/04.Reference/13.identity-and-access/docs.en.md]: Let's add a link to Keystone API documentation. Maybe with a table of supported API versions.

File: pages/04.Reference/13.identity-and-access/docs.en.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+OpenStack's Identity API is implemented via Keystone. It provides API client authentication, service discovery, and distributed multi-tenant authorization.
+
+## Password
+
+The common way for a user or service to authenticate towards Keystone is by using username and password. You may enter the user credentials in the OpenStack dashboard, or use them inside of your software to manage your OpenStack ressources. A sample OpenStack RC file which can be used for the OpenStack python CLI client would be the following.
+
+```shell
```
Comment (stefreak) [inline pages/04.Reference/13.identity-and-access/docs.en.md]: I believe we have a How-To or tutorial for that? if yes maybe we link it from here.

In the reference guide I would instead expect a short description of the Authentication flow (e.g. Retrieve token from Keystone, send request to Nova with the token, Nova checks the token using keystone, Nova sends response) and a table with supported methods. In that table we should also mention that we don't support two factor authentication yet.

File: pages/04.Reference/13.identity-and-access/docs.en.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+If you want to revoke a token before it expires we may do so manually.
+
+```shell
+openstack token revoke ab4d1e500c5245e5a166892943118a44
+```
+
+## Roles
+
+By default a user has 2 roles, the viewer role which represents a read-only access and the operator role which represents a read-write access. It is necessary for the user to have both roles assigned in order to be able to delegate the read-only or read-write rights when creating application credentials.
```
Comment (stefreak) [inline pages/04.Reference/13.identity-and-access/docs.en.md]: Maybe we add a section about user access management (the cloudsupport team manages users and groups etc) and then a table with the supported roles.

File: pages/04.Reference/13.identity-and-access/docs.en.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+export OS_INTERFACE=public
+export OS_ENDPOINT_TYPE=public
+export OS_USERNAME="<your-username>"
+export OS_PASSWORD="<your-password>"
+export OS_INTERFACE="public"
+export OS_REGION_NAME="dbl"
+#export OS_REGION_NAME="cbk"
+```
+
+## Tokens
```
[the remark this answers is not in the document]
Comment (lgrebe-sys) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: We have a tutorial for application credentials waiting here : https://github.com/syseleven/grav-docs-syseleven-stack/pull/64.

File: pages/04.Reference/13.identity-and-access/docs.en.md (hunk 1 above)
Comment (lgrebe-sys) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: Alright, I will leave this one out

File: pages/04.Reference/13.identity-and-access/docs.en.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (stefreak) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: Can you put both in one merge request, so you can cross-link the reference guide and the tutorial?

[the remark this answers is not in the document]
Comment (lgrebe-sys) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: sure thing

[the remark this answers is not in the document]
Comment (stefreak) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: So then let's change this to a section like authentication methods, and list all the methods, linking to tutorials if necessary?

File: pages/04.Reference/13.identity-and-access/docs.en.md (hunk 2 above)
Comment (lgrebe-sys) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: I will link the documentation, well there are API v2 and API v3 for identity. We can list the supported API versions for all components if you mean that

File: pages/04.Reference/13.identity-and-access/docs.en.md (hunk 3 above)
Comment (lgrebe-sys) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: So we describe : https://docs.openstack.org/keystone/queens/api_curl_examples.html

File: pages/04.Reference/13.identity-and-access/docs.en.md (hunk 4 above)
Comment (lgrebe-sys) [reply pages/04.Reference/13.identity-and-access/docs.en.md]: Will take care

**Pull request merged**